app/views/submissions/view.html.erb

```diff
@@ -52,6 +52,32 @@
       var headerPositionStr = null;
     <% end %>
     hljs.initHighlightingOnLoad();
+    document.addEventListener('DOMContentLoaded', (event) => {
```

Comment:
can you add a comment explaining exactly what you are doing here? It is a little hard to understand

Reply:
okay